sh/tests/file-details_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+challenge() {
+    submitted=$(cd "$1" && bash "$script_dirS"/student/file-details.sh)
+    expected=$(cd "$1" && bash "$script_dirS"/solutions/file-details.sh)
+
+    diff <(echo "$submitted") <(echo "$expected")
+}
+
+challenge hard-perm
```

Comment:
hard-perm folder is not included in the pr, I didn't found it in the repo

Reply:
Yes but it will be because it was added in the hard perm exercise